### PR TITLE
adding secret info link in logs

### PR DESCRIPTION
**Pull request by SumiranAgg**

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Login to Azure subscription
 name: 'Azure Login'
-description: 'Azure Login wraps the az login, allowing Azure actions to log into Azure or to run Az CLI scripts.'
+description: 'Azure Login wraps the az login, allowing Azure actions to log into Azure or to run Az CLI scripts. github.com/Azure/Actions'
```
Comment (niadak) [inline action.yml]: Why this change?

Comment (SumiranAgg) [reply action.yml]: Request by @N-Usha so we can show more details about our action in marketplace
